### PR TITLE
Creation of D2D Server List PR for Jakarta

### PR DESCRIPTION
**Pull request by marktrayer**

### CODE REVIEW

File: swagger2.0/oic.r.d2dserverlist.swagger.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+               "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+            },
+            "if": {
+               "description": "The OCF Interface set supported by this Resource",
+               "type": "array",
+               "minItems": 2,
+               "uniqueItems": true,
+               "items": {
+                  "type": "string",
+                  "enum": ["oic.if.baseline", "oic.if.rw"]
```
Comment (WAvdBeek) [inline swagger2.0/oic.r.d2dserverlist.swagger.json]: do rw interface first

Comment (marktrayer) [reply swagger2.0/oic.r.d2dserverlist.swagger.json]: Fixed. See latest commit.

**Review by WAvdBeek — APPROVED** [review]

File: swagger2.0/oic.r.d2dserverlist.swagger.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                  "enum": ["oic.if.rw", "oic.if.baseline"]
+               },
+               "readOnly": true
+            },
+            "dis": {
+               "description": "This Property maintains the list of D2D Device's Device UUID",
+               "type": "array",
+               "uniqueItems": true,
+               "items": {
+                  "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
```
Comment (mkettrick) [inline swagger2.0/oic.r.d2dserverlist.swagger.json]: I don't think this is the correct reference.  It should point to a UUID definition, not the "id" definition.

Comment (marktrayer) [reply swagger2.0/oic.r.d2dserverlist.swagger.json]: Fixed. Please see latest commit.

File: swagger2.0/oic.r.d2dserverlist.swagger.json [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+                        "b63ece84-f904-11ea-bf17-0c9d928536d7"
+                     ]
+                  },
+                  "schema": { "$ref":"#/definitions/D2DServerList-dis" }
+               },
+               "400": {
+                  "description": "UPDATE request without Device UUID parameter"
+               },
+               "404": {
+                  "description": "There is no Device that has requested Device UUID"
```
Comment (joochlee) [inline swagger2.0/oic.r.d2dserverlist.swagger.json:67]: can we change "description" as follows ? :
"There is no Device for the requested Device UUID parameter"

Comment (marktrayer) [reply swagger2.0/oic.r.d2dserverlist.swagger.json:67]: Fixed. See latest commit.

**Pull request closed without merging**